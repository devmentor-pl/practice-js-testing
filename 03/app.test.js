import randomNumber from './app';

test('returns 1 when the range is from 1 to 1', () => {
    const showNumber = randomNumber(1, 1);
    expect(showNumber).toBe(1);
});

test('return a radnom number in the range', () => {
    const min = 3
    const max = 4
    const number = randomNumber(min, max)

    expect(number).toBe(4)

})

test('throws an error when argument is not a number', () => {
    function generateRandomItem() {
        randomNumber("5");
    }

    expect(generateRandomItem).toThrow();
});