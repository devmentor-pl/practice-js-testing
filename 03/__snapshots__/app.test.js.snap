// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`throws an error when argument is not a number 1`] = `"Arguments must be numbers"`;
